pkg/kapis/devops/v1alpha2/register_test.go

```diff
@@ -0,0 +1,206 @@
+package v1alpha2
+
+import (
+	"context"
+	"github.com/emicklei/go-restful"
+	"github.com/jenkins-zh/jenkins-client/pkg/core"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	fake4 "k8s.io/client-go/kubernetes/fake"
```

Comment:
fake2, fake3 and fake4 are incomprehensible here. Please rename it to proper name.

Reply:
Good catch!. Fixed it.